WP_Auth0.php

```diff
@@ -140,6 +140,46 @@ public static function ready() {
 		return true;
 	}
 
+	/**
+	 * Get the tenant region based on a domain.
+	 *
+	 * @param string $domain Tenant domain.
+	 *
+	 * @return string
+	 */
+	public static function get_tenant_region( $domain ) {
```

Comment:
Could you just regexp for this so not hardcoded checks?

Reply:
That's not a bad idea, actually ... would make this more future-proof if we add additional regions. I'll add, rebase, and re-submit.

Reply:
@cocojoe - Much better with regex and can handle new regions 👍 thanks for the suggestion. Tests added.